pkg/restapi/v1/issuer/controller_test.go

```diff
@@ -944,3 +1225,138 @@ func TestController_PostIssuerProfilesProfileIDDeactivate(t *testing.T) {
 	})
 }
 */
+
+func Test_validateIssueCredOptions(t *testing.T) {
+	type args struct {
+		options *IssueCredentialOptions
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    []crypto.SigningOpts
+		wantErr bool
+	}{
+		// TODO: Add test cases.
```

Comment:
is it going to be added in follow-up tickets? do we have the one created?

Reply:
I totally spaced it out. Added.